Collection.h

```diff
@@ -0,0 +1,24 @@
+//
+//  Collection.h
+//  Trippy
+//
+//  Created by Catherine Lu on 7/7/22.
+//
+
+#import <Foundation/Foundation.h>
+#import "Location.h"
+#import "Parse/Parse.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface Collection : NSObject
```

Comment:
This name is a bit on the generic side... maybe something like `LocationCollection`?